### DISCUSSION TITLE
Interested in contributing to the organization for GSOC'23

### DISCUSSION THREAD

**Discussion by Anu-123-gif** [koordinates/kart#754] [Google Soc / SoD]
I am Anubhuti Shruti, a student from India with an interest in Programming. I am familiar with python, flask, JAVA and HTML and up for learning technologies specific to different projects.

I would like to know about the different ways to contribute to the organization.

Thanks!

**Comment by rcoup** [comment]
Hi, we haven't decided whether to participate in GSoC 2023 yet... (and if we do, whether we'll be accepted) — check back at the end of February

**Comment by Anu-123-gif** [comment]
Hey,
Thanks for replying. I understand the situation. Would still love to know
the different ways to contribute to the organization.

Thanks a lot,
Anubhuti.
